### PR TITLE
added new game Whack a mole

### LINKED ISSUE #4475: [New game]: Whack-a-Mole
### 🎮 Game Request

Game logic and basic description:- In this game, moles will pop up randomly from holes, and the player has to click on them to score points. The game will be timed, and the player will have a limited amount of time to score as many points as possible.

### Point down the features

Game points

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by 1911aditi**
## PR Description 📜

i have added a new game whack a mole.
pls assign me under GSSOC'24.
Fixes #4475 

<hr>
 
##4475

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

whack a mole

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @1911aditi ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by 1911aditi** [comment]
I have resolved the issue
Can u check now?

**Comment by 1911aditi** [comment]
Can u check now?

File: README.md [hunk 1]
```diff
@@ -329,13 +329,25 @@ This repository also provides one such platforms where contributers come over an
 | [Sudoku_light_theme](https://github.com/kunjgit/GameZone/tree/main/Games/Sudoku_light_theme) |
 | [Find_the_ball](https://github.com/kunjgit/GameZone/tree/main/Games/Find_the_ball) |
 |[Color The Page](https://github.com/kunjgit/GameZone/tree/main/Games/Color_The_Page)|
+
+
+|[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game)   |
```
Comment (ishita-43) [inline README.md:334]: remove the duplicate entry

File: README.md [hunk 2]
```diff
@@ -329,13 +329,25 @@ This repository also provides one such platforms where contributers come over an
...
+| [Snake](https://github.com/kunjgit/GameZone/tree/main/Games/snake)                               |
```
Comment (ishita-43) [inline README.md:335]: remove this entry

File: README.md [hunk 3]
```diff
@@ -329,13 +329,25 @@ This repository also provides one such platforms where contributers come over an
...
+| [whack a mole](https://github.com/kunjgit/GameZone/tree/main/Games/whack%20a%20mole)             |
+
+|[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game) |
```
Comment (ishita-43) [inline README.md:338]: remove this entry

File: README.md [hunk 4]
```diff
@@ -329,13 +329,25 @@ This repository also provides one such platforms where contributers come over an
...
+| [Snake](https://github.com/kunjgit/GameZone/tree/main/Games/snake)                |
```
Comment (ishita-43) [inline README.md:339]: remove this entry

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make the desired changes!!

**Comment by 1911aditi** [comment]
check now

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
@1911aditi resolve merge conflicts

**Comment by 1911aditi** [comment]
check now

**Comment by kunjgit** [comment]
Thank you  @1911aditi , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**